lib/jekyll/strapi/collection.rb

```diff
@@ -20,27 +20,35 @@ def generate?
       def each
         # Initialize the HTTP query
         path = "/#{@config['type'] || @collection_name}?_limit=10000"
-        uri = URI("#{@site.endpoint}#{path}")
-        Jekyll.logger.info "Jekyll Strapi:", "Fetching entries from #{uri}"
+        uri = URI("#{@site.endpoint}/api#{path}")
         # Get entries
-        response = Net::HTTP.get_response(uri)
+        Jekyll.logger.info "Jekyll Strapi:", "Fetching entries from #{uri}"
+        # response = Net::HTTP.get_response(uri)
+        response = strapi_request(uri)
         # Check response code
         if response.code == "200"
-          result = JSON.parse(response.body, object_class: OpenStruct)
+          _result = JSON.parse(response.body, object_class: OpenStruct)
+          result = _result.data 
         elsif response.code == "401"
           raise "The Strapi server sent a error with the following status: #{response.code}. Please make sure you authorized the API access in the Users & Permissions section of the Strapi admin panel."
         else
           raise "The Strapi server sent a error with the following status: #{response.code}. Please make sure it is correctly running."
         end
-
         # Add necessary properties
         result.each do |document|
           document.type = collection_name
           document.collection = collection_name
           document.id ||= document._id
+          document.title = document.attributes.Title
```

Comment:
should the T in Title be capitalized? I noticed you used document.title in site.rb.

Reply:
Well spotted, and perhaps this will be removed forever. Need to check more api structure, but I think it will be redundant.